src/services/LocalStorageService/index.test.js

```diff
@@ -0,0 +1,40 @@
+import LocalStorageService from '.';
+
+describe('LocalStorage Service', () => {
+	beforeEach(() => {
+		Storage.prototype.getItem = jest.fn();
+		Storage.prototype.setItem = jest.fn();
+		Storage.prototype.removeItem = jest.fn();
+	});
+
+	afterEach(() => {
+		jest.resetAllMocks();
+	});
+
+	describe('getAppData', () => {
+		it('should return stored app data from LocalStorageService.APP_DATA_KEY', () => {
+			const data = { foo: 'bar' };
+			const expectedOutput = JSON.stringify(data);
+			localStorage.getItem.mockReturnValue(expectedOutput);
+
+			const result = LocalStorageService.getAppData();
+			expect(localStorage.getItem).toHaveBeenCalledWith(LocalStorageService.APP_DATA_KEY);
```

Comment:
There is no need to check local storage calls in this case. The fact that we call `localStorage.getItem` is current implementation. Let's test `getAppData` and other function as a black-box.

Reply:
Done.